marketdata/rest.go

```diff
@@ -88,6 +88,9 @@ func NewClient(opts ClientOpts) *Client {
 var DefaultClient = NewClient(ClientOpts{})
 
 func defaultDo(c *Client, req *http.Request) (*http.Response, error) {
+	goVer, moduleVer := alpaca.GetVersion()
+	req.Header.Set("User-Agent", "APCA-GO/"+moduleVer+"/"+goVer)
```

Comment:
why not encode formatted str in version.go?

Reply:
Addressed: https://github.com/alpacahq/alpaca-trade-api-go/pull/254/commits/0a55aca0ed5eac63f4259b7f71116c04385650b8